client/app/styles/onboarding.styl

```diff
@@ -20,7 +20,7 @@
 
 
 main
-    max-width: (600/16)em
+    width: 30em
```

Comment:
`main` here is a flex item. Forcing `width`causes responsive issues. We should prefer `flex: 0 1 30em`, so `main` won't grow above `30em`, but will properly fot in small screens.